flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/logical/SplitAggregateRule.scala

```diff
@@ -138,9 +138,11 @@ class SplitAggregateRule extends RelOptRule(
     val windowProps = fmq.getRelWindowProperties(agg.getInput)
     val isWindowAgg = WindowUtil.groupingContainsWindowStartEnd(agg.getGroupSet, windowProps)
     val isProctimeWindowAgg = isWindowAgg && !windowProps.isRowtime
+    // TableAggregate is not supported. see also FLINK-21923.
```

Comment:
I do not find any explanation why TableAggregate is not supported in FLINK-21923

Reply:
I left a message here [FLINK-21923](https://issues.apache.org/jira/browse/FLINK-21923?focusedCommentId=17315334&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17315334).
If it is TableAggregate, ClassCastException will definitely occur